src/Akeneo/Pim/Enrichment/Component/Product/Validator/UniqueAxesCombinationSet.php

```diff
@@ -51,17 +51,18 @@ public function addCombination(EntityWithFamilyVariantInterface $entity, string
     {
         $familyVariantCode = $entity->getFamilyVariant()->getCode();
         $parentCode = $entity->getParent()->getCode();
+        $loweredAxisValueCombination = \mb_strtolower($axisValueCombination);
 
-        if (isset($this->uniqueAxesCombination[$familyVariantCode][$parentCode][$axisValueCombination])) {
-            $cachedIdentifier = $this->uniqueAxesCombination[$familyVariantCode][$parentCode][$axisValueCombination];
+        if (isset($this->uniqueAxesCombination[$familyVariantCode][$parentCode][$loweredAxisValueCombination])) {
+            $cachedIdentifier = $this->uniqueAxesCombination[$familyVariantCode][$parentCode][$loweredAxisValueCombination];
             if ($cachedIdentifier !== $entity->getIdentifier()) {
                 if ($entity instanceof ProductInterface) {
                     throw new AlreadyExistingAxisValueCombinationException(
                         $cachedIdentifier,
                         sprintf(
                             'Variant product "%s" already have the "%s" combination of axis values.',
                             $cachedIdentifier,
-                            $axisValueCombination
+                            $loweredAxisValueCombination
```

Comment:
In my opinion we should keep the case sent by the user in order to be really clean, that's was we have in master and I think it's better
```suggestion
                            $axisValueCombination
```